app/views/admin/move_advisers/choose_subsidiary.html.erb

```diff
@@ -0,0 +1,27 @@
+<%= form_for(@form, url: confirm_admin_move_advisers_path, method: :get) do |f| %>
+  <h1>Choose subsidiary</h1>
+  <% if @form.errors.any? %>
+    <ul class="t-errors bg-danger">
+      <% @form.errors.full_messages.each do |message| %>
+        <li><%= message %></li>
+      <% end %>
+    </ul>
+  <% end %>
+
+  <div class="form-group">
+  <%= f.collection_radio_buttons(:to_firm_id, @form.subsidiaries, :id,
+      ->(firm) { "(#{firm.fca_number}) #{firm.registered_name}" }) do |b| %>
+    <div class="t-subsidiary radio">
+      <%= b.label(class: 't-subsidiary-label') { b.radio_button(class: 't-subsidiary-field') + b.text } %>
+    </div>
+  <% end %>
+  </div>
+
+  <div class="t-hidden-fields form-group">
+    <% @form.adviser_ids.each do |adviser_id| %>
+      <%= f.hidden_field(:adviser_ids, multiple: true, value: adviser_id, class: 't-adviser') %>
+    <% end %>
+    <%= f.hidden_field(:to_firm_fca_number, value: @form.to_firm_fca_number, class: 't-to_firm_fca_number') %>
+    <%= f.submit('Next', class: 't-next') %>
+  </div>
```

Comment:
Does this submit button belong in this element? `t-hidden-fields` seems incorrect if so.